### PR TITLE
update philanthropy page + footer

### PR DESCRIPTION
**Pull request by bellesea**

### CODE REVIEW

File: pages/philanthropy/index.js [hunk 1]
```diff
@@ -298,37 +299,59 @@ const Philanthropy = ({ posts = [] }) => {
...
+              <Text
+                sx={{ mt: 1, display: 'block', opacity: 0.8 }}
+                fontSize={2}
+                color="white"
+              >
+                Your contribution is tax-deductible.
+                <br />
+                Hack Club is a 501(c)(3) nonprofit with the EIN 81-2908499.
+              </Text>
+            </Fade>
```
Comment (sampoder) [inline pages/philanthropy/index.js:382]: What if we adapted the header on the current `/donate`?

File: components/footer.js [hunk 2]
```diff
@@ -109,8 +109,8 @@ const Footer = ({ dark = false, children, ...props }) => (
           <NextLink href="/press" passHref>
             <Link>Press Inquiries</Link>
           </NextLink>
-          <NextLink href="/donate" passHref>
-            <Link>Donate</Link>
+          <NextLink href="/philanthropy" passHref>
+            <Link>Support us</Link>
```
Comment (sampoder) [inline components/footer.js]:
```suggestion
            <Link>Support Us</Link>
```

File: pages/philanthropy/index.js (hunk 1 above)
Comment (bellesea) [reply pages/philanthropy/index.js:382]: omg duh! yes that's a great idea! thank you!

**Comment by bellesea** [comment]
@sampoder cool if i merge first then open more PRs for future edits?

**Comment by sampoder** [comment]
Cool with me!

**Pull request merged**